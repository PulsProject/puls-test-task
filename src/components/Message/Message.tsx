import React from 'react';
import { Box, Typography } from '@mui/material';

const Message = () => {
  return (
        <Box>
            <Typography>There is nothing to show here!</Typography>
        </Box>
  );
};

export default Message;