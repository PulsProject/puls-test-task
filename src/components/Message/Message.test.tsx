import React from 'react';
import { render } from '@testing-library/react';
import Message from './Message';

jest.mock('../../theme/theme', () => jest.fn());
jest.mock('../../data/loans.json', () => jest.fn(() => ({ loanRequests: [] })));

describe('<Message />', () => {
  it('it should show "There is nothing to show here!" message', () => {
    const { container } = render(<Message />);
    expect(container.firstChild).toMatchSnapshot();
  });
});