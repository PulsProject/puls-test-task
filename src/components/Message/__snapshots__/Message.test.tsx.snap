// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> it should show "There is nothing to show here!" message 1`] = `
<div
  class="MuiBox-root css-0"
>
  <p
    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    There is nothing to show here!
  </p>
</div>
`;
